### ISSUE TITLE
Android: `SDL_GetAndroidCachePath()` crashes

### ISSUE THREAD

**Issue by Sackzement** [libsdl-org/SDL#11297]
Calling the function `SDL_GetAndroidCachePath()` on my android phone causes the application to crash.

specifically at this line:
https://github.com/libsdl-org/SDL/blob/e9b5f6749e47388c9283c78102f2b088bb39d829/src/core/android/SDL_android.c#L2404
where `CallObjectMethod()` is called with a `mid` value of `NULL`.

My phone specs:
Model name: Samsung Galaxy Grind Prime Plus
Model number: SM-G532F
Android(/SDK) version 6(6.0.1) (API level 23)

**Comment by madebr** [comment]
https://developer.android.com/reference/android/content/Context#getCacheDir()
Looks like 
https://github.com/libsdl-org/SDL/blob/e9b5f6749e47388c9283c78102f2b088bb39d829/src/core/android/SDL_android.c#L2403
should be
```c
                                  "getCacheDir", "()Ljava/io/File;");
```

Can you try this?

**Comment by Sackzement** [comment]
Yes, it works!
Thanks!

**Comment by madebr** [comment]
Thanks for testing!
I pushed the fix as 0bec18245

**Comment by Sackzement** [comment]
Thanks for fixing! :)
